### ISSUE TITLE
If tabs aren't find, automatically convert multiple spaces to tabs

### ISSUE THREAD

**Issue by ftyers** [jonorthwash/ud-annotatrix#107]
Sometimes people copy/paste from things like piratepad or pastebin where tabs aren't maintained... they are converted to spaces. If there are no tabs in the input, then we should convert series of more than one space into tabs.

**Comment by ftyers** [comment]
Fixed.

**Comment by jonorthwash** [comment]
Fixed in which revisions?

**Comment by ftyers** [comment]
Here is a file that isn't working:

```
# sent_id = bremaik.vislcg.txt:181:6085
# text = Disklêriet en deus ar pastor 81 bloaz e oa Iwerzhon an Hanternoz o voulc'hañ ur maread a beoc'h.
# text[fra¹] = Il a déclaré le pasteur 81 ans il était l'Irlande le Nord en train d'entamer une époque de paix.
# labels = to_check
1       Disklêriet      disklêriañ      VERB    vblex   Tense=Past|VerbForm=Part        0       root    _       _
2       en deus kaout   VERB    vblex   Gender=Msc|Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin        1       aux     _       _
3       ar      an      DET     det     _       4       det     _       _
4       pastor  pastor  NOUN    n       Gender=Msc|Number=Sing  1       nsubj   _       _
5       81      81      NUM     num     Number=Plur     6       nummod  _       _
6       bloaz   bloaz   NOUN    n       Gender=Msc|Number=Sing  4       nmod:gen        _       _
7       e       e       PART    vpart   _       13      aux     _       _
8       oa      bezañ   VERB    vblex   Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin   13      aux     _       _
9       Iwerzhon        Iwerzhon        PROPN   np      Number=Sing     13      nsubj   _       _
10      an      an      DET     det     _       11      det     _       _
11      Hanternoz       hanternoz       NOUN    n       Gender=Msc|Number=Sing  9       nmod:gen        _       _
12      o       o       PART    vpart   _       13      aux     _       _
13      voulc'hañ       boulc’hañ       VERB    vblex   VerbForm=Inf    1       ccomp   _       _
14      ur      un      DET     det     _       15      det     _       _
15      maread  maread  NOUN    n       Gender=Msc|Number=Sing  13      obj     _       _
16      a       a       ADP     pr      _       17      case    _       _
17      beoc'h  peoc’h  NOUN    n       Gender=Msc|Number=Sing  15      nmod    _       _
18      .       .       PUNCT   sent    _       1       punct   _       _
```

**Comment by ftyers** [comment]
```
$ git diff -r HEAD
diff --git a/standalone/lib/annotator.js b/standalone/lib/annotator.js
index 6a528c9..1b33ded 100644
--- a/standalone/lib/annotator.js
+++ b/standalone/lib/annotator.js
@@ -507,10 +507,10 @@ function drawTree() {
     // If there are >1 CoNLL-U format sentences is in the input, treat them as such
     // conlluMultiInput(newContent); // TODO: move this one also inside of this func, and make a separate func for calling them all at the same time 
 
-    // if(newContent != content) {
-    //     content = newContent;
-    //     $("#indata").val(content);
-    // }
+    if(newContent != content) {
+        content = newContent;
+        $("#indata").val(content);
+    }
     // -- to be moved out -- 
 
     conlluDraw(content);
```

**Comment by ftyers** [comment]
Fixed in 126a40e
